### ISSUE TITLE
Crash when changing resource packs

### ISSUE THREAD

**Issue by auctumnus** [MehVahdJukaar/Supplementaries#5]
With Forge 35.1.36, on Minecraft 1.16.4, and Supplementaries 0.9.20c, I get a crash when switching texture packs. [Full crash log here.](http://ix.io/2MaC)

**Comment by auctumnus** [comment]
Looks like the issue is in [this line](https://github.com/MehVahdJukaar/Supplementaries/blob/3e337d4eb85fe61b5c09ac57759efe7681fa68ba/src/main/java/net/mehvahdjukaar/supplementaries/events/RightClickEvent.java#L180):
```java
else if(ServerConfigs.cached.THROWABLE_BRICKS && (i.isIn(Tags.Items.INGOTS_BRICK)||i.isIn(Tags.Items.INGOTS_NETHER_BRICK))){
    event.getToolTip().add(new TranslationTextComponent("message.supplementaries.throwable_brick").mergeStyle(TextFormatting.GRAY));
}
```
I'm not experienced with either java or forge, but this might need to either wait to register this event until the tags are fully bound, or just do a simple check to see if the tag is ready before adding this?

**Comment by MehVahdJukaar** [comment]
yeah Just added a null check there. I've got another report that said the same

**Comment by MehVahdJukaar** [comment]
I have to say that I still can't replicate it. I've posted a beta version on my discord so you can tell me if it's still not working

**Comment by auctumnus** [comment]
Issue seems to be fixed with 0.9.21, thanks!
